aim/web/ui/src/modules/BaseExplorer/components/Controls/BoxProperties/BoxProperties.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Tooltip } from '@material-ui/core';
+import { IBaseComponentProps } from 'modules/BaseExplorer/types';
+
+import ControlPopover from 'components/ControlPopover/ControlPopover';
+import { Icon } from 'components/kit';
+
+import BoxPropertiesPopover from './Popover';
+
+import { IBoxPropertiesProps, IBoxConfigState } from '.';
+
+function BoxProperties(props: IBoxPropertiesProps) {
+  const {
+    engine,
+    engine: {
+      useStore,
+      boxConfig: { stateSelector: boxSelector },
+      controls: {
+        boxProperties: { settings },
+      },
+    },
+  } = props;
+  const boxConfig: IBoxConfigState = useStore(boxSelector);
```

Comment:
What about rename this to boxProperties?

Reply:
This is the name selected to use inside engine. There will be some changes for naming there, with the cleanup phase.
Changing the name inside engine will cause a lot of code changes, since there are function names based on the property name, lets keep this in enhancements list, and decide the best name while cleaning up the code base.